src/main/scala/is/hail/io/plink/PlinkLoader.scala

```diff
@@ -200,8 +137,59 @@ object PlinkLoader {
            |  Duplicate IDs: @1""".stripMargin, duplicateIds)
     }
 
-    val vds = parseBed(hc, bedPath, ids, annotations, signature, variants, nPartitions, a2Reference)
-    vds
-  }
+    val results = bedPaths.zip(bimPaths).map { case (bedPath, bimPath) =>
+      val variants = parseBim(bimPath, hc.hadoopConf, a2Reference)
+      val nVariants = variants.length
+      if (nVariants <= 0)
+        fatal(s".bim file does not contain any variants [$bimPath]")
```

Comment:
I think the format of `.bim file does not contain any variants: $bimPath` is more consistent with our other error messages, but I don't care too much either way.